test/client_test.rb

```diff
@@ -374,73 +374,4 @@
       refute_same $client.connection, client.connection
     end
   end
-
-  describe "automatic retry of requests" do
-    before do
-      @events = []
-      @subscriber = ActiveSupport::Notifications.subscribe do |*args|
-        @events << ActiveSupport::Notifications::Event.new(*args)
-      end
-    end
-
-    after do
-      @events.clear
-      ActiveSupport::Notifications.unsubscribe(@subscriber)
-    end
-
-    it "defaults to no retries" do
-      stub_request(:get, $client.url+"/_cat/indices").
-        to_timeout.then.
-        to_return({
-          headers: {"Content-Type" => "text/plain; charset=UTF-8"},
-          body: "green open test-index 1 0 0 0 159b 159b"
-        })
-
-      assert_raises(ElastomerClient::Client::ConnectionFailed) {
-        $client.get("/_cat/indices")
-      }
-    end
```

Comment:
We probably want to keep this test. That way we can keep ensuring it has no retries by default.

Reply:
I added this back!